assets/sass/_normalize-overrides.scss

```diff
@@ -145,6 +145,10 @@ dd {
   margin-left: 0;
 }
 
+dd + dd {
+  @include margin(-24, "top");
```

Comment:
Negative margins can result in unanticipated overlays, is there a better way of achieving the desired effect?